app/web/features/auth/locales/en.json

```diff
@@ -280,6 +280,30 @@
           "change": "The gender displayed on your profile is changed"
         }
       }
+    },
+    "push_notifications": {
+      "title": "Push Notifications",
+      "description": "You can enable push notifications for Couchers.org on your device. We are working on expanding this feature.",
+      "allow_push": "Click the 'Allow' button in your browser popup to enable push notifications.",
+      "enabled_message": "You currently have push notifications <1>enabled</1>.",
+      "disabled_message": "You currently have push notifications <1>disabled</1>.",
+      "error_generic": "Error during subscription to push notifications",
+      "error_unsupported": "Push notifications or Service Workers are not supported in this browser.",
+      "permission_denied": {
+        "platform_settings_description": "Still having issues after enabling notifications in your browser? You may need to adjust your platform settings.",
+        "instructions": {
```

Comment:
We will also need to add instructions for iOS (which requires pinning the app to the home screen): I think it returns "denied" without popping up the modal at all in general.

But I think we can leave that for later, I'd like to get this merged in soon :)

Reply:
I added a quick `isMobile` check and a generic message for now. Can go back and add better ones later.